src/lsstseries/ensemble.py

```diff
@@ -15,15 +15,25 @@ class Ensemble:
     def __init__(self, token=None, client=None, **kwargs):
         self.result = None  # holds the latest query
         self.token = token
-        self._data = None
+
+        self._source = None  # Source Table
+        self._object = None  # Object Table
```

Comment:
It is possible to crash a bunch of the functions below by creating an `Ensemble` and then not loading anything into it (so both tables are `None`).  I've gone back and forth one whether it would be worth adding checks for this at the start of those functions, but I think the errors you would see in that case are reasonable already (trying to assign something to `None`).

Reply:
Yeah, I see both sides to this as well. My hope is that this is really just a documentation issue, where repeated examples of seeing `ens = Ensemble()` followed immediately by a loader function would dissuade users from trying to do any data operation on the empty Ensemble.